tools/tokenserver/util.py

```diff
@@ -0,0 +1,58 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file was adapted from [this file](https://github.com/mozilla-services/tokenserver/blob/master/tokenserver/scripts/__init__.py) on the legacy Tokenserver